docs/source/user_guide/automl.ipynb

```diff
@@ -539,7 +539,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "automl_auto = AutoMLSearch(X_train=X, y_train=y, problem_type='binary')\n",
+    "automl_auto = AutoMLSearch(X_train=X, y_train=y, problem_type='binary', _automl_algorithm='iterative')\n",
```

Comment:
Fixed the docs here by calling iterative but will properly add more content and fix the docs in #2693.